sdk/storage/azure-storage-blob-cryptography/pom.xml

```diff
@@ -98,6 +98,12 @@
       <version>1.0.8</version> <!-- {x-version-update;com.azure:azure-identity;dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-security-keyvault-keys</artifactId>
+      <version>4.2.0-beta.5</version> <!-- {x-version-update;com.azure:azure-security-keyvault-keys;current} -->
```

Comment:
This should target `dependency` correct?

Reply:
Yeah I tried that but I think dependency is latest GA version, which doesnt have the latest LocalKeyEncryptionKey stuff

Reply:
Ah, didn't realize that.